app/code/Magento/QuoteGraphQl/Model/Cart/GetCartForUser.php

```diff
@@ -84,6 +84,16 @@ public function execute(string $cartHash, ?int $customerId): Quote
                 )
             );
         }
+
+        if ($customerId) {
```

Comment:
Looks like we already load cart at
https://github.com/magento/graphql-ce/blob/2363896edb0a8d7707a2f71862cf8d0de6423f4a/app/code/Magento/QuoteGraphQl/Model/Cart/GetCartForUser.php#L65

We should reuse `\Magento\Quote\Model\Quote::getIsActive` method after loading

Reply:
PR was updated. Test was added